concourse/update-content/script.bash

```diff
@@ -13,25 +13,8 @@ set -ex
 
 yarn
 
-export ARCHIVE_URL="https://archive.cnx.org"
-
-node script/entry update-content-versions
-
-mv src/config.books.js src/config.books.new.js
-git checkout src/config.books.js
-
-book_ids=$(node -e "$(cat <<script
-  const oldBooks = require('./src/config.books.js');
-  const newBooks = require('./src/config.books.new.js');
-  Object.keys(newBooks).forEach((key) => {
-    if (newBooks[key].defaultVersion !== oldBooks[key].defaultVersion) {
-      console.log(key);
-    }
-  });
-script
-)")
-
-rm src/config.books.new.js
+approved_books=$(curl -sL https://github.com/openstax/content-manager-approved-books/raw/master/approved-books.json)
```

Comment:
Could/Should this look up the default branch? `default_branch` in https://api.github.com/repos/openstax/content-manager-approved-books